docs/paypal.md

```diff
@@ -0,0 +1,169 @@
+# Pay with PayPal
+
+Accept PayPal payments in your Android app using the PayPal Payments SDK.
+
+## How it works
+
+This diagram shows how your client, your server, and PayPal interact:
+
+// TODO - Get a diagram of the payment flow similar to [this](https://developer.paypal.com/braintree/docs/start/overview#how-it-works)
+
+## Eligibility
```

Comment:
For PayPal eligibility, we could copy what we have here in the Checkout docs: https://developer.paypal.com/sdk/in-app/#link-eligibility

Reply:
Hmm we could but will we be offering a web fallback to allow all PayPal eligible merchants to use the SDK (beyond Europe and North America)?

Reply:
Yep, other regions should have a successful web fallback. Not sure if we'd want to call that out here.